_posts/2024-12-01-18.1.0-release.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Apache Arrow 18.1.0 Release"
+date: "2024-12-01 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 18.1.0 release. This
+covers roughly one month of development work and includes [**15 resolved
+issues**][1] on [**25 distinct commits**][2] from [**8 distinct
+contributors**][2]. See the [Install Page](https://arrow.apache.org/install/) to
+learn how to get the libraries for your platform.
+
```

Comment:
```suggestion

The primary motivation for this minor release was to address packaging issues
for Python, Java, C#, R and MATLAB which caused binaries for those platforms to
report a version of `18.0.0-SNAPSHOT` instead of `18.0.0`. The affected binaries
were otherwise built correctly. See [this mailing list
post](https://lists.apache.org/thread/t8k7l2hsbgdt7cszj7hrpjdfpn91n5zb) for more
details.

```

@kou how do you feel about calling this out like I've done above? I'm totally okay not including it but I'm expecting the rest of the post to be quite empty and that may lead to confusion as to why we did a minor release.

Reply:
Thanks! +1 for this suggestion!